packages/app/package.json

```diff
@@ -28,7 +28,7 @@
     "dist": "npm run build && electron-builder --publish never",
     "TEST COMMANDS": "=========================================================",
     "test": "cross-env TZ=UTC jest --config=jest.config.js",
-    "test:watch": "npm test -- --watch",
+    "test:watch": "npm test -- --watch --runInBand",
```

Comment:
I've used it as you said for debugging. I've encountered a warning with suggestion to turn this flag on so it can provide me with additional output about errors.